Automattic/jetpack
# Social Logos: update to include new Facebook logo design

Fixes #15954


#### Changes proposed in this Pull Request:

* This PR updates our social icon font and the svg, bringing the changes from this PR:
https://github.com/Automattic/social-logos/pull/94/

The new logo is a bit off center in sharing buttons in my opinion, but I'm no designer :)

**Before**

![image](https://user-images.githubusercontent.com/426388/85848335-48629e00-b7a9-11ea-9a5d-933b4d5a097f.png)

![image](https://user-images.githubusercontent.com/426388/85848347-4d275200-b7a9-11ea-8b81-377d193e9f95.png)

![image](https://user-images.githubusercontent.com/426388/85848361-50224280-b7a9-11ea-8ee1-a95ad72d217f.png)

![image](https://user-images.githubusercontent.com/426388/85851737-8bc00b00-b7af-11ea-8c5d-01fc8608f616.png)



**After**

![image](https://user-images.githubusercontent.com/426388/85848442-6defa780-b7a9-11ea-8d7e-4a3c8f6ee1b4.png)

![image](https://user-images.githubusercontent.com/426388/85848446-70ea9800-b7a9-11ea-8452-a56f46311885.png)

![image](https://user-images.githubusercontent.com/426388/85848460-75af4c00-b7a9-11ea-9af1-1b3119d87b56.png)

![image](https://user-images.githubusercontent.com/426388/85851743-8fec2880-b7af-11ea-9381-bf605c43a1c7.png)



#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* On a new site, go to Jetpack > Settings and activate both the Sharing feature and the extra sidebar widgets
* View a post, and check the Facebook sharing button at the bottom of that post.
* Go to Settings > Sharing in your dashboard
* Change the sharing button style, and check that the new Facebook logo looks good in all styles
* Go to Appearance > Customize
* Add a Social Icons widget, and add a Facebook URL in there.
* Make sure the icon in the widget looks good.

#### Proposed changelog entry for your changes:

* Social Logos: update Facebook Logo design
